packages/schematics/angular/application/index.ts

```diff
@@ -153,12 +153,13 @@ function addAppToWorkspaceFile(options: ApplicationOptions, appDir: string): Rul
 
   if (options.inlineTemplate === true
```

Comment:
Nit: can also remove `=== true` from here 😉

Reply:
Okay, I changed that as well. I hesitated to do so since I didn't want to change the existing code.